charts/backstage/values.yaml

```diff
@@ -99,6 +99,10 @@ serviceAccount:
 
 securityContext: {}
 
+## Configure backstage to connect to your local cluster
+kubernetesPlugin:
```

Comment:
This only enables local cluster connection should we rename this to reflect that it is in fact only the cluster it is deployed to ?